src/renderer/components/NewServerModal.tsx

```diff
@@ -428,6 +480,105 @@ class NewServerModal extends React.PureComponent<Props, State> {
                         {this.getServerNameMessage()}
                         {this.getServerURLMessage()}
                     </div>
+                    {this.props.editMode &&
+                        <>
+                            <hr/>
+                            <h5>
```

Comment:
Is this semantically an h5 or are you using it for styling purposes? For accessibility reasons (screen readers and whatnot) if it's not an h5 you should change it to something else.